graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java

```diff
@@ -14,37 +14,37 @@
 /**
  * Base interface representing an abstract project model with a project name, version, type, and location on disk. Projects may be source-based or
  * binary based.
- * 
+ *
  * Additional models may extend this to support additional project types (eg, Maven-based projects).
- * 
+ *
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
 @TypeValue(ProjectModel.TYPE)
 public interface ProjectModel extends WindupVertexFrame
 {
-    public static final String TYPE = "ProjectModel";
-    public static final String DEPENDENCY = "dependency";
-    public static final String PARENT_PROJECT = "parentProject";
-    public static final String ROOT_FILE_MODEL = "rootFileModel";
-    public static final String PROJECT_MODEL_TO_FILE = "projectModelToFile";
+    public static final String TYPE = "Project";
+    public static final String PREFIX = "Project:";
+    public static final String DEPENDENCY = PREFIX + "dependency";
+    public static final String PARENT_PROJECT = PREFIX + "parentProject";
+    public static final String ROOT_ORIGIN_LOCATION = PREFIX + "rootOriginLocation";
+    public static final String CONTAINED_FILES = PREFIX + "containedFiles";
     public static final String SOURCE_BASED = "sourceBased";
-    public static final String DESCRIPTION = "description";
-    public static final String ORGANIZATION = "organization";
-    public static final String URL = "url";
-    public static final String NAME = "name";
-    public static final String VERSION = "version";
-    public static final String PROJECT_TYPE = "projectType";
+    public static final String DESCRIPTION = PREFIX + "description";
+    public static final String ORGANIZATION = PREFIX + "organization";
+    public static final String URL = PREFIX + "url";
+    public static final String NAME = PREFIX + "name";
+    public static final String VERSION = PREFIX + "version";
+    public static final String PROJECT_TYPE = PREFIX + "type";
 
     /**
-     * This represents the root directory (in the case of a source-based analysis) or root archive (for binary analysis) containing this particular
-     * project.
-     * 
+     * This represents the root directory (in the case of a source-based analysis)
+     * or root archive (for binary analysis) containing this particular project.
      */
-    @Adjacency(label = ROOT_FILE_MODEL, direction = Direction.OUT)
-    void setRootFileModel(FileModel fileModel);
+    @Adjacency(label = ROOT_ORIGIN_LOCATION, direction = Direction.OUT)
```

Comment:
What would be better? RootFile is wrong because what's a root file for a file that came from a war in an ear? Is it the EAR archive? Or it's unzipped dir? Is it the WAR archive or it's unzipped dir? And what is it for a source project?

RootOriginLocation is clearer, because it indicates that it's the file from which the project originated, not the unzipped. RootFile is ambiguous.